docs/tutorials/go.md

```diff
@@ -705,12 +705,11 @@ The response contains a `base64` encoded representation of the data we submitted
 To get the original value out of this data, we can use the `base64` command line utility:
 
 ```bash
-echo cm9ja3M=" | base64 -d
+echo "cm9ja3M=" | base64 -d
 ```
 
 ## Outro
 
 I hope everything went smoothly and your first, but hopefully not the last,
 CometBFT application is up and running. If not, please [open an issue on
-Github](https://github.com/cometbft/cometbft/issues/new/choose). To dig
-deeper, read [the docs](https://docs.cometbft.com/v0.37/).
+Github](https://github.com/cometbft/cometbft/issues/new/choose).
```

Comment:
Do we handle these kinds of multi-line links correctly now in the docs rendering? Or does this still cause the links to break?

Reply:
I've tested and this link works because it's not pointing to an `.md` file which would need to be parsed by the relative link plugin. But thanks for bringing this up.